package.json

```diff
@@ -3,6 +3,7 @@
   "version": "16.0.1",
   "description": "A responsive and accessible date range picker component built with React",
   "main": "index.js",
+  "module":"esm/index.js",
```

Comment:
I'm a very very strong -1 for this change. This is a non-standard package.json field, and the fact that bundlers support it is highly toxic to the ecosystem.

Every time a package ships a "module" field, it makes it that much harder for a standard package.json field to ever exist.

Reply:
I agree! But tools like webpack and rollup still look up for "module". So IMO this will atleast reduce the bundle size for millions of consumer today, until the `.mjs` becomes standard and enabled in node by default?

Reply:
No amount of performance or bundle size is worth polluting the ecosystem, imo.